scripts/openstack/manifest-tests/bootstrap-flavor/test_bootstrap_flavor.py

```
@@ -0,0 +1,99 @@
+import glob
+import yaml
+
+ASSETS_DIR = ""
+
+
+class BootstrapFlavorMachine(unittest.TestCase):
+    def setUp(self):
+        """Parse the OpenStackMachine manifests and install-config into Python data structures."""
+        # Read install-config from cluster-config.yaml to get expected flavors.
+        with open(f'{ASSETS_DIR}/manifests/cluster-config.yaml') as f:
+            cluster_config = yaml.load(f, Loader=yaml.FullLoader)
```

Ruff (0.15.17)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)